test/e2e/common/pods.go

```diff
@@ -530,17 +530,13 @@ var _ = framework.KubeDescribe("Pods", func() {
 					{
 						Name:    "main",
 						Image:   "gcr.io/google_containers/busybox:1.24",
-						Command: []string{"/bin/sh", "-c", "echo container is alive; sleep 600"},
+						Command: []string{"/bin/sh", "-c", "echo container is alive; sleep 10000"},
 					},
 				},
 			},
 		}
 
 		By("submitting the pod to kubernetes")
-		defer func() {
```

Comment:
The test namespaces will get cleaned up automatically. So I don't see a concern. In-fact it is preferable to not auto-delete unless deletion is part of the test's goals.